com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -321,6 +321,7 @@ private unsafe void SendAnimStateClientRpc(AnimationMessage animSnapshot, Client
         }
 
         [ClientRpc]
+        // Server wants to forward a trigger for a client to play / reset
```

Comment:
please use the standard `/// ` xmldoc format :)

Reply:
Done! :)